src/hotspot/share/classfile/classLoaderData.cpp

```diff
@@ -258,7 +258,7 @@ bool ClassLoaderData::ChunkedHandleList::contains(oop p) {
 
 #ifndef PRODUCT
 bool ClassLoaderData::ChunkedHandleList::owner_of(oop* oop_handle) {
-  Chunk* chunk = _head;
+  Chunk* chunk = Atomic::load_acquire(&_head);
```

Comment:
It's unclear to me if this is actually concurrently accessed, but if we want to be conservative and add it here, then that's fine. However, then I'd also like to see an Atomic::load of chunk->_size.